cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -914,6 +923,12 @@ pub mod pallet {
 	#[pallet::storage]
 	pub type CustomValidationHeadData<T: Config> = StorageValue<_, Vec<u8>, OptionQuery>;
 
+	/// The claim queue offset.
+	/// This value is expected to be mostly a constant. It's used by collators to determine when to
```

Comment:
It is used by both validators and collators to determine the core index and the relay  block when this needs to be backed on chain.,

Reply:
Would be a good idea to mention that this is going to be used in the `UMPSignal::SelectCore` message.